packages/cli/src/installers/node/index.ts

```diff
@@ -3,51 +3,25 @@ import { spawnSync } from "child_process"
 import chalk from "chalk"
 import inquirer from "inquirer"
 import { validatePushApiKey } from "@appsignal/core"
+import * as fs from "fs"
+import * as path from "path"
 
 import { SUPPORTED_NODEJS_INTEGRATIONS } from "../../constants"
 import { spawnDemo } from "../../commands/demo"
 
 /**
- * This is the very last thing to be displayed by the installer CLI! The
- * indendation is intentional, due to JavaScripts handling of multi-line
- * strings
+ * Installs the Node.js integration in the current working process.cwd().directory
  */
-const displayOutroMessage = (pushApiKey: string, name: string) => `
-🎉 ${chalk.greenBright(
-  "Great news!"
-)} You've just installed AppSignal to your project!
-
-The next step is adding your Push API key to your project. The best way to do this is with an environment variable:
-
-${chalk.bold(`export APPSIGNAL_PUSH_API_KEY="${pushApiKey}"`)}
-
-If you're using a cloud provider such as Heroku etc., seperate instructions on how to add these environment variables are available in our documentation:
-
-🔗 https://docs.appsignal.com/nodejs/configuration
-
-Then, you'll need to initalize AppSignal in your app. Please ensure that this is done in the entrypoint of your application, ${chalk.cyan(
-  "before all other dependencies are imported!"
-)}
-
-${chalk.bold(`const { Appsignal } = require("@appsignal/nodejs");
-
-const appsignal = new Appsignal({
-  active: true,
-  name: "${name}"
-});`)}
-
-Some integrations require additional setup. See https://docs.appsignal.com/nodejs/integrations/ for more information.
-
-Need any further help? Feel free to ask a human at ${chalk.bold(
-  "support@appsignal.com"
-)}!
-`
-
-/**
- * Installs the Node.js integration in the current working directory
- */
-export async function installNode(pkg: { [key: string]: any }) {
+export async function installNode(pkg: { [key: string]: any }, dir: string) {
   const cwd = process.cwd()
+  const src = path.join(dir, "src")
+  let configurationFilename: string
+
+  if (fs.existsSync(src)) {
```

Comment:
Maybe also check that it's a folder, since we intend to write a file to it? I know, it would be weird if it wasn't...

Reply:
Added an extra check in https://github.com/appsignal/appsignal-javascript/pull/579/commits/ca1eaceae8244ec100e5e58bfa1d1dce7b3ccfe5.